scripts/generateAbi.js

```diff
@@ -1,19 +1,26 @@
 const fs = require("fs");
 const solc = require("solc");
 
+// Assuming each contract is in the file with the same name.
+var contracts = ["IPyth", "AbstractPyth", "MockPyth"];
```

Comment:
I suggest documenting the ABI generation process in the readme in case someone adds a new contract later.

Reply:
you can do this in a separate PR if you would like.

Reply:
Will do now, so this be complete.